channels/chan_echolink.c

```diff
@@ -1820,6 +1822,25 @@ static void send_info(const void *nodep, const VISIT which, const int depth)
 			ast_log(LOG_WARNING, "Exceeded buffer size");
 			return;
 		}
+
+		i += j;
+
+		if (instp->mymessage[0] != '\0') {
+			char *p = instp->mymessage;
+			while ((p = strstr(p, "\\n")) != NULL) {
+				*p = '\n';  // Replace the literal \n with a newline character
```

Comment:
The [Asterisk coding guidelines ](https://docs.asterisk.org/Development/Policies-and-Procedures/Coding-Guidelines/) include :
```
* Don't use C++ type (//) comments.
```
and we're trying to follow those conventions.

Reply:
This should be resolved.